DSharpPlus/Entities/BaseDiscordMessageBuilder.cs

```diff
@@ -39,12 +40,39 @@ public T SuppressNotifications()
         return (T)this;
     }
 
+    /// <summary>
+    /// Enables or disables support for V2 components; MESSAGES WITH V2 COMPONENTS CANNOT BE DOWNGRADED.
+    /// </summary>
+    /// <param name="enable"></param>
+    /// <returns></returns>
+    public T EnableV2Components(bool enable = true)

```

Comment:
very sceptical of this parameter

Reply:
I initially removed it, but then realized that clear components doesn't disable the flag; so added it back (calling .Clear() does remove the flag); if you'd like to enforce that API usage, I can remove it.

Reply:
an analogous `DisableV2Components` would be preferable to this parameter; `Clear()` would also be fine but we should be prepared to get support questions